### ISSUE TITLE
"Tools" under remote Desktop non-functioning

### ISSUE THREAD

**Issue by mschubeck** [Ylianst/MeshCentral#141]
I'm not sure if these are reserved for future use and just not implemented yet, or whether I have something misconfigured.

In the bottom right corner, the following behaviors exist for each of the tools there:
![image](https://user-images.githubusercontent.com/48728727/54721876-9108c180-4b39-11e9-9a4a-d5e1c60c549e.png)
1. The "Tools" button shows active processes that I can terminate. This works successfully, but not sure if that is all that should be available.
2. The "Open a web address on remote computer" appears to do nothing - on either side.
3. The "Display a notification on the remote computer" appears to do nothing.
4. The "Open a chat window on remote computer" prompts the remote user to open a web browser but does not say what it's for.  It appears to ignore the default browser settings on the host computer.  Edit: I see it opens a new tab.  I would expect some type of notice that the remote user at the host machine has closed the browser/MeshMessenger application.

I presume these are a work in progress, I am just looking for a confirmation on this.  In my scenario above, I'm calling the machine I'm remoting **into** as the "host," and the machine I'm remoting **from** the "remote."

**Comment by PathfinderNetworks** [comment]
I just tested the Open a web address on remote computer, display notification on remote computer, and open a chat window on remote computer.  All three worked.  The display a notification just uses the Windows 10 (not sure about 7 as I haven't tested on that) notification bubble that pops above the system tray.  The chat feature does use a web page for the chat- so it does need to pop open a website for it to work.

**Comment by mschubeck** [comment]
Update: After Pathfinder's post, I looked at this some more and it seems it's "miraculously" working for me all of a sudden.  Even though I tested this behavior on multiple machines yesterday, today every one of those machines is working properly.  So, I looked at my MC2 host server and found that it was actually in the middle of auto-installing updates while I was testing this yesterday, and it ultimately rebooted overnight.  At this point, I'm assuming that the other activities the server was going through were interfering at the same time that I was testing this.

Sorry for the confusion.
